nx.json

```diff
@@ -114,15 +114,22 @@
     "./tools/src/debug/debug.plugin.ts",
     {
       "plugin": "./tools/src/npm/npm.plugin.ts",
-      "options": { "verbose": true }
+      "options": {
+        "verbose": true
+      }
     },
     {
       "plugin": "./tools/src/publish/publish.plugin.ts",
-      "options": { "verbose": true }
+      "options": {
+        "verbose": true
+      }
     },
     {
       "plugin": "./tools/src/verdaccio/verdaccio.plugin.ts",
-      "options": { "verbose": true }
+      "options": {
+        "verbose": true
+      }
     }
-  ]
+  ],
+  "useInferencePlugins": false
```

Comment:
What does it do?

Reply:
Leftover from failed migration https://nx.dev/concepts/inferred-tasks